### ISSUE TITLE
Can't import ambf_client

### ISSUE THREAD

**Issue by darthandvader** [WPI-AIM/ambf#218]
I've sourced the ambf/build/devel folder and can find the path in my bash file but still when I use the python script in pycharm htere's still "No module named ambf_client". I also tried switching to python 2.7 interpreter but it didn't work.

I looked at the issue #9 and #67 it seems like they can launch the client after adding the path so I'm comfused which step I missed.

Thank you!

**Comment by adnanmunawar** [comment]
Hello, 

Are you running Pycharm via the terminal with your `.bashrc` file sourced?

**Comment by darthandvader** [comment]
Hi, I run Pycharm directly from application interface. Should I run it from terminal instead?

**Comment by adnanmunawar** [comment]
Yes, you should run it from a terminal with the ROS and AMBF sourced prior to launch.

**Comment by darthandvader** [comment]
You are right, ambf_client module can be detected when I launch pycharm from terminal. But there's a new problem is that "Cannot find reference 'Client' in '__init__.py' ". I installed gym with python 3.8 and also installed ros noetic. I don't know if I missed something.

**Comment by adnanmunawar** [comment]
What is this `init.py` file? Is that from AMBF?
